src/http/AuthHttpClient.js

```diff
@@ -206,6 +206,19 @@ class AuthHttpClient {
         return this._instance(request);
     }
 
+    async triggerStepUpNotification(stepUp) {
+        const type = stepUp.tokenId ? 'tokenStepUp' : 'requestStepUp';
```

Comment:
FWIW, I don't understand RequestStepUps and could use an example or a test or something. StepUp/tokenStepUp makes sense, though.

Reply:
asked vivek and it's for user SCA under PSD2, like every once in a while it will notify you that you need to SCA again to view your balance, transactions, etc, at least that's my understanding. gonna merge this now so everyone can test and potentially add tests later.

Reply:
Article 10 page 16: http://ec.europa.eu/finance/docs/level-2-measures/psd2-rts-2017-7782_en.pdf

Reply:
Ah, thank you. That link is darned informative.